data-model/SDP-data-model-templates/textual-artifact_sdp-template.json

```diff
@@ -30,52 +44,76 @@
     {
       "id": "URI",
       "type": "CV",
-      "note": [
-        "array of strings"
-      ],
-      "associated_dates": [
+      "associated_date": [
         {
+          "date_type": "CV",
           "date_human_readable": "",
           "date_normalized": "ISO-8601 Dates",
           "note": [
             "array of strings"
           ]
         }
       ],
-      "associated_entities": [
+      "associated_person": [
         {
-          "entity_type": "CV",
-          "role": "CV",
-          "id": "URI/ARK",
+          "relation": "CV",
+          "id": "URI",
           "note": [
             "array of strings"
           ]
         }
       ],
+      "associated_place": [
+        {
+          "relation": "CV",
+          "id": "URI",
+          "note": [
+            "array of strings"
+          ]
+        }
+      ],
+      "note": [
+        "array of strings"
+      ],
       "sources": [
-        "ARK/ID of a Paratext? URI of a Zotero Bibl?"
+        "Paratext ARKs | Zotero URI(?)"
       ]
     }
   ],
-  "paratexts": [
-    {
-      "id": "ARK",
-      "type": "CV",
-      "locus": "",
-      "transcription": {
-        "$langCode": "Note: lang code refers to the language of transcribed text. transcription object may have multiple $langCode fields."
-      },
-      "translation": {
-        "$langCode": "Note: lang code refers to the language of translated text. translation object may have multiple $langCode fields."
-      },
-      "note": [
-        ""
-      ],
-      "attested_entities": {
+  "paratexts": {
+    "rubric": [
```

Comment:
I added this list of top-level types to demonstrate, and based on what we are getting in Airtable. More than happy to add, edit, or delete.

Reply:
Note that the list is "rubric", "prologue", "incipit", "explicit", "subscription", and "marginalia". The paratexts field ends on line 554